pyaedt/modeler/Primitives3D.py

```diff
@@ -138,10 +138,13 @@ def create_cylinder(self, cs_axis, position, radius, height, numSides=0, name=No
         --------
         >>> from pyaedt import Hfss
         >>> aedtapp = Hfss()
-        >>> ret_object = aedtapp.modeler.primitives.create_cylinder(cs_axis='Z', position=[0,0,0], radius=2, height=3,
-        ...                                                name="mycyl", matname="vacuum")
+        >>> cylinder_object = aedtapp.modeler.primitives.create_cylinder(cs_axis='Z', position=[0,0,0],
+        ...                                                              radius=2, height=3, name="mycyl",
+        ...                                                              matname="vacuum")
 
         """
+        assert radius > 0, "Radius must be greater than 0."
```

Comment:
Raise rather than assert. See https://github.com/pyansys/pyaedt/pull/686/files#r777120264

```suggestion
        if radius < 0:
            raise ValueError("Radius must be greater than 0.")
```